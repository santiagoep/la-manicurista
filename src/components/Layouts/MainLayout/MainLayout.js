import React from "react";

import Logo from "@components/Icons/Logo";
import childrenProps from "@utils/common-propTypes";
import Accessibility from "@components/Accessibility/Accessibility";
import { StyledContainer, StyledTitleContainer } from "./MainLayout.styled";

const MainLayout = ({ children }) => (
  <StyledContainer>
    <Accessibility />
    <StyledTitleContainer>
      <Logo />
    </StyledTitleContainer>
    {children}
  </StyledContainer>
);

MainLayout.propTypes = {
  children: childrenProps.isRequired,
};

export default MainLayout;
