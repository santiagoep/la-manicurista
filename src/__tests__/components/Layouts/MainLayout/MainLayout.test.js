import React from "react";

import render from "../../../render";
import MainLayout from "@components/Layouts/MainLayout/MainLayout";

describe("<MainLayout />", () => {
  it("should render snapshot", () => {
    const { renderer } = render({ theme: true, redux: true });
    const component = renderer(
      <MainLayout>
        <h1>Test</h1>
      </MainLayout>
    );
    expect(component).toMatchSnapshot();
  });
});
