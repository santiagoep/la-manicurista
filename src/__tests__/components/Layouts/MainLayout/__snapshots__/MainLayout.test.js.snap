// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainLayout /> should render snapshot 1`] = `
<main
  className="MainLayoutstyled__StyledContainer-yzzxwr-0 jLgpgd"
>
  <span
    className="Accessibilitystyled__StyledContainer-gi4yzg-0 dWjzqH"
  >
    <div
      className="Accessibilitystyled__StyledItem-gi4yzg-1 jkNukM accessibility__font-size"
      onClick={[Function]}
    >
      Font size:md
    </div>
    <div
      className="Accessibilitystyled__StyledItem-gi4yzg-1 jkNukM accessibility__theme"
      onClick={[Function]}
    >
      Tema:LIGHT
    </div>
    <div
      className="Accessibilitystyled__StyledItem-gi4yzg-1 jkNukM accessibility__language"
      onClick={[Function]}
    >
      Idioma:es
    </div>
  </span>
  <div
    className="MainLayoutstyled__StyledTitleContainer-yzzxwr-1 ciPhaR"
  >
    <img
      alt="LaManicurista Logo"
      className="Logo__StyledImage-sc-17vxmqw-0 gCRnca"
      src="https://lamanicurista.com/wp-content/uploads/2019/01/Logo-La-manicurista-1.png"
    />
  </div>
  <h1>
    Test
  </h1>
</main>
`;
